vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md

```diff
@@ -19,7 +18,6 @@ The steps below walk you though installing the Entando platform in an EKS cluste
 - Configure an IAM role to allow kubernetes to manage the cluster
 - Create an EKS cluster with 5 nodes (to allow expansion for microservices)
 - Install nginx as an ingress controller in the cluster
-- Register a domain (if you don't already have one) and configure it for wildcard subdomains.
 - Install Entando
 
```

Comment:
Would take out the period. The list does not have periods.

Reply:
This line was deleted so should be all set.